_data/faq.yml

```diff
@@ -458,46 +458,59 @@
-                    The additional data is stored on SD card by default (in a folder called ".cgeo").
+                    You may store the database of c:geo either system internally or in a directory on your user storage.
+                    
+                    You can change the database location in Menu → Settings → System → "Database location". \\
+                    As the database availability is critical for using c:geo and the typical size is only some megabyte, we decided not to allow moving it to the external SD card.
+                    If you enable the option "On user storage" the database will be moved to the folder /Android/data/cgeo.geocaching/files/databases (may vary depending on your device).
 
-                Some users are complaining, that the additional data as well as the database (if external storage is selected) are still stored in the internal memory of their devices.
-                We always place the folder on the storage medium that the device provides as SD card. On some devices, this storage medium is part of the internal memory, which is mounted as an SD card (It's like an SD card fixed mounted in your device).
+                2. The additional data (e.g. images, maps) for all the caches stored in the database:
 
-                Unfortunately there is no standardized method to find the location of a real SD card on your device. There are some workarounds but those might fail on some devices, thus we decided not to implement them for now.
-                Furthermore starting with Android 4.4 write access to the real SD card is forbidden for apps.
+                    This additional data can be much larger as it contains the images of the cache description and (optionally) the images retrieved from logs and (optionally) static maps. \\
+                    By default this data is stored on the internal user storage in /Android/data/cgeo.geocaching/files/GeocacheData/.
+                    
+                    You can move this data to your external SD card in Menu → Settings → System → "Geocache Data Directory". \\
+                    If your device supports using the external SD card this menu option will show you the available target directories as well as the current size of your geocache data and the available space on the target directories.
+                    
+                    Please consider the folllowing when you decide to store the geocache data on your external SD card:
+                    
+                    - The initial moving of the geocache data might take a long time depending on the amount of data to be moved.
+                      Please don't interrupt the process and make sure your device is sufficiently charged.
+                    - If you remove the external SD card or the SD card is full the additional geocache data will no longer be available and c:geo might no longer work as expected.
```

Comment:
Same as above. I tried to be very generic, to avoid regular changes to FAQ and just say If you do somethign wrong, it will happen somethgin unexpected. This could either be an error message or a crash.